modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -20,12 +22,24 @@ class ClaimsAndAppealsController < ApplicationController
         end
       end
 
-      before_action(except: %i[index get_claim request_decision]) do
+      before_action(only: %i[get_appeal]) do
         authorize :evss, :access?
```

Comment:
Appeals don't go through EVSS, I probably would just get rid of this `before_action` block

Reply:
It looks like our controller is still using an evss proxy for `get_appeal`

Reply:
The `Mobile::V0::Claims::Proxy` i.e. `evss_claims_proxy` uses Caseflow for appeals and EVSS for claims. If this is for the `get_appeal` call specifically, it doesn't seem to make sense to utilizing the EVSS Pundit policy

Reply:
Jerek is correct. We shouldn't be putting appeals behind an evss or LH policy

Reply:
Thanks for the clarification Jerek, I'm not good with understanding the branching upstream services

Reply:
It should be behind this policy: `app/policies/appeals_policy.rb`, but you should double check with @cadibemma since her ticket is related to the claims and appeals policies

Reply:
Looks like that PR is updating it so I think we should be covered there

Reply:
@jperk51 @jerekshoe so does that mean I should leave this alone since the linked pr will update the policy?

Reply:
I think it's safe to leave the policy stuff for the other PR. Just my opinion though